io.github.sameboy.SameBoy.yml

```diff
@@ -0,0 +1,50 @@
+app-id: io.github.sameboy.SameBoy
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm13
+build-options:
+  append-path: /usr/lib/sdk/llvm13/bin
+  prepend-ld-library-path: /usr/lib/sdk/llvm13/lib
+command: sameboy
+rename-desktop-file: sameboy.desktop
+rename-icon: sameboy
+finish-args:
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --device=dri
+  - --filesystem=home
+
+modules:
+  - name: rgbds
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: git
+        url: https://github.com/gbdev/rgbds.git
+        tag: v0.5.2
+        commit: 9d993d84e85eeb8dd304e48463d89865795e97ed
+
+  - name: sameboy
+    buildsystem: autotools
+    no-autogen: true
+    make-args:
+      - DATA_DIR=${FLATPAK_DEST}/share/sameboy/
+    make-install-args:
+      - PREFIX=${FLATPAK_DEST}
+      - DATA_DIR=${FLATPAK_DEST}/share/sameboy/
+    post-install:
+      # https://github.com/flatpak/flatpak-xdg-utils/issues/35
+      - install -Dm 644 FreeDesktop/sameboy.desktop ${FLATPAK_DEST}/share/applications/sameboy.desktop
+      - install -Dm 644 io.github.sameboy.SameBoy.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+    sources:
+      - type: git
```

Comment:
It would make sense to configure extra-data-checker if it's not going to be maintained by upstream.